### DISCUSSION TITLE
Should the `endpoint` calls return related data (similar to `lookup` and `LookupResponse`)?

### DISCUSSION THREAD

**Discussion by MitchTurner** [free-honey/naumachia#156] [Ideas]
For example, with Fortuna, if a caller submits a valid solution and mints the block TUNA, it would be useful to the miner to get back the info from the subsequent block so the miner could start mining that block preemtively. That being said, maybe we don't want to because it is very possible that our solution won't be accepted and the subsequent block will be different. It's also possible to form that information without Naumachia's help?

The point still stands that it might be useful to have that information! I'm not completely convinced though.

**Comment by MicroProofs** [comment]
"it would be useful to the miner to get back the info from the subsequent block so the miner could start mining that block preemptively" 
I would say this is wanted since the only drawback is you have to start over if a different solution is accepted which is the normal case anyway
